src/Aspire.Hosting.Azure.Functions/AzureFunctionsProjectResourceExtensions.cs

```diff
@@ -236,6 +237,64 @@ public static IResourceBuilder<AzureFunctionsProjectResource> WithReference<TSou
         });
     }
 
+    /// <summary>    
+    /// </summary>
+    /// <param name="builder"></param>
+    /// <returns></returns>
+    public static IResourceBuilder<AzureFunctionsProjectResource> PublishWithAdminKey(this IResourceBuilder<AzureFunctionsProjectResource> builder)
```

Comment:
Do we anticipate that end-users will need to call this API in their code (or overloads of it that target different key types)?